packages/core/src/test/testUtil.ts

```diff
@@ -109,6 +109,10 @@ export async function createTestWorkspace(
          * the subDir where the workspace folder will point to within the temp folder
          */
         subDir?: string
+        /**
+         * overridden file name, to allow testing for file extensions
+         */
+        fileName?: string
```

Comment:
just like the other parameters, there is no need to mention "override". this is no different than the other parameters.

```suggestion
         * optional filename suffix
         */
        fileNameSuffix?: string
```